.gitignore

```diff
@@ -3,7 +3,12 @@ dist/
 
 # Jest
 coverage/
+
+# Environment files
 .integration.test.env
+.integration.test-dev.env
```

Comment:
You can use a wildcard here e.g. `.integration.*.env`

Reply:
Done.